scripts/ci.js

```diff
@@ -129,9 +174,17 @@ const main = () => {
 };
 
 if (require.main === module) {
-  main().catch(err => {
```

Comment:
SUGGESTION: Without top-level `await`, I personally think `main().catch(err => {})` reads a lot better and more intuitively.

Reply:
Maybe something like:

```js
main()
  .catch(err => {
    console.error(err);
    process.exit(1); // eslint-disable-line no-process-exit
  })
  .then(() => {
    // CloudFormation waitFor may still be running so force an exit
    process.exit(0); // eslint-disable-line no-process-exit
  });
```

Reply:
... but I'm cool either way.